src/bit-systems/object-menu.ts

```diff
@@ -98,7 +113,7 @@ export function objectMenuSystem(world: HubsWorld, sceneIsFrozen: boolean, useri
   ObjectMenu.targetRef[menu] = objectMenuTarget(world, menu, sceneIsFrozen);
   if (ObjectMenu.targetRef[menu]) {
     moveToTarget(world, menu);
-    handleClicks(world, menu);
+    handleClicks(world, menu, hubChannel);
   }
   render(world, menu, sceneIsFrozen);
```

Comment:
Minor, and not part of this PR but calling this `render` is confusing in this context since its not actually rendering anything. I realize you were using it more in the React sense than the Three sense, but we are in engine code here so my brain defaults to the later. In the camera tool system I called the similar function `updateUI`. In this case maybe its just `updateButtonVisibility`

Reply:
done